lib/linguist/generated.rb

```diff
@@ -588,7 +588,8 @@ def generated_graphql_relay?
     def generated_gamemakerstudio?
       return false unless ['.yy', '.yyp'].include? extname
       return false unless lines.count > 3
-      return lines[2].match(/\"modelName\"\:\s*\"GM/)
+      return lines[2].match(/\"modelName\"\:\s*\"GM/) ||
+             lines[0] =~ /^\d\.\d\.\d/\u2190/
```

Comment:
Syntax error. That's what's causing the build to fail:

~~~
/home/travis/build/github/linguist/lib/linguist/generated.rb:592: syntax error, unexpected ')'
t.match(/\d\.\d\.\d/\u2190.*/)
~~~


Escape or delete this forward-slash:

~~~
t.match(/\d\.\d\.\d/\u2190.*/)
                   ^~~~~~~~~~~
~~~

Reply:
Yep, I was working on it and letting travis run the tests ^_^